Sources/Helpers/OASettingsHelper.mm

```diff
@@ -195,9 +242,11 @@ - (void) onSettingsCollectFinished:(BOOL)succeed empty:(BOOL)empty items:(NSArra
         if (pluginIndependentItems.count > 0)
         {
             // TODO: add ui dialogs as in Android
-//            FragmentManager fragmentManager = activity.getSupportFragmentManager();
-//            ImportSettingsFragment.showInstance(fragmentManager, pluginIndependentItems, file);
-            [self importSettings:_importTask.getFile items:_importTask.getItems latestChanges:@"" version:1];
+            //            FragmentManager fragmentManager = activity.getSupportFragmentManager();
+            //            ImportSettingsFragment.showInstance(fragmentManager, pluginIndependentItems, file);
+            UIViewController* incomingURLViewController = [[OAImportProfileViewController alloc] initWithItems:pluginIndependentItems];
```

Comment:
Why do you cast it to UIViewController here? Why would you leave the commented out android code if you added the correct behavior?